changelog.asc

```diff
@@ -2,6 +2,21 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.18.1 (20240523)
+* Minor fixes to polish up the 2.18.0 release, main notes below 👇
+* AnkiDroid is participating in Google Summer of Code again!
+* Your https://opencollective.com/ankidroid[donations] inspire a new generation of open source contributors 💓
+* Fix crash in TagsDialog (this is for you, Anking users!)
+* Fresh translations from our community of https://crowdin.com/project/ankidroid[translators] (thank you!)
+* Fix language handling for regional variants
+* Fix crash in template previewer
+* Fix deck handling note editor
+* Fix reviewer regressions in javascript and visual performance
```

Comment:
Needs 24h to see whether this lands in the release

Reply:
based on findings that it fails for you on 2.18.0 already I'm going to leave it in and send 2.18.1 out on beta to soak